app/Resources/views/composer/install.html.twig

```diff
@@ -0,0 +1,74 @@
+{% macro command(command) %}
+    <div class="orange">
+        <p>&#36; {{ command }}</p>
+    </div>
+{% endmacro %}
+
+{% import _self as Helper %}
+
+<div class="window">
+    <div class="titlebar">
+        <div class="buttons">
+            <div class="close">
+                <a class="closebutton" href="#"><span><strong>x</strong></span></a>
+            </div>
+            <div class="minimize">
+                <a class="minimizebutton" href="#"><span><strong>&ndash;</strong></span></a>
+            </div>
+            <div class="zoom">
+                <a class="zoombutton" href="#"><span><strong>+</strong></span></a>
+            </div>
+        </div>
+        ezplatform@eZ-Systems: ~/www
+    </div>
+    <div class="content">
+        {% set dbhost = "127.0.0.1" %}
```

Comment:
Git nazi here: If we'll need it, then we'll add it 😈 
Please remove those lines, they're useless now.